src/main/java/jp/pay/net/LivePayjpResponseGetter.java

```diff
@@ -507,7 +508,15 @@ private static PayjpResponse getMultipartPayjpResponse(
 	}
 
 	private static void handleAPIError(String rBody, int rCode) throws InvalidRequestException, AuthenticationException, CardException, APIException {
-		Error error = APIResource.GSON.fromJson(rBody, ErrorContainer.class).error;
+		Error error = null;
+		try {
+			error = APIResource.GSON.fromJson(rBody, ErrorContainer.class).error;
+		} catch (JsonSyntaxException e) {
+			throw new APIException("Not json response.", rCode, rBody, e);
```

Comment:
`Not json response.` が有益な情報か微妙なので、rBodyを見てもらったほうが良さそう
```suggestion
			throw new APIException(rBody, rCode, e);
```